packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/evaluate-expression.ts

```diff
@@ -22,6 +22,13 @@ export interface EvaluateExpressionProps extends sfn.TaskStateBaseProps {
    * @default lambda.Runtime.NODEJS_18_X
    */
   readonly runtime?: lambda.Runtime;
+
+  /**
+   * This is the role that will be assumed by the function upon execution.
+   *
+   * @default a role will be automatically created
```

Comment:
```suggestion
   * @default - a role will be automatically created
```

We need to add a `-` after the `@default` tag if we're using a sentence after instead of specifying the default value.